handler/src/main/java/io/netty/handler/ssl/OpenSsl.java

```diff
@@ -410,14 +410,12 @@ private static void loadTcNative() throws Exception {
         Set<String> libNames = new LinkedHashSet<String>(4);
         // First, try loading the platform-specific library. Platform-specific
         // libraries will be available if using a tcnative uber jar.
-        libNames.add("netty-tcnative-" + os + '-' + arch);
+        libNames.add("netty_tcnative-" + os + '-' + arch);
         if (LINUX.equalsIgnoreCase(os)) {
             // Fedora SSL lib so naming (libssl.so.10 vs libssl.so.1.0.0)..
-            libNames.add("netty-tcnative-" + os + '-' + arch + "-fedora");
+            libNames.add("netty_tcnative-" + os + '-' + arch + "-fedora");
         }
         // finally the default library.
-        libNames.add("netty-tcnative");
-        // in Java 8, statically compiled JNI code is namespaced
         libNames.add("netty_tcnative");
 
         NativeLibraryLoader.loadFirstAvailable(SSL.class.getClassLoader(),
```

Comment:
Talked with @Scottmitch offline... this is not related to this pr and we will do a followup.